src/_scss/pages/search/searchSidebar.scss

```diff
@@ -8,37 +8,79 @@
     box-shadow: $container-shadow;
     border-radius: rem(5);
     margin-top: rem(32);
+
     .sidebar-header {
         display: none;
-        @include media($tablet-screen) {
-            @include display(flex);
-            @include align-items(center);
-            @include flex-flow(row);
+        @include media($small-screen) {
+            display: inline-table;
+            @include align-items(left);
             @include justify-content(flex-start);
             position: relative;
             border-bottom: rem(1) solid $color-gray-lighter;
             height: rem(70);
+            width: 100%;
             margin: 0;
             padding: 0 0 0 rem(15);
-            & .sidebar-title {
-                @include h6;
-                @include flex(1 1 auto);
-                color: $color-gray;
-                font-size: rem(19);
-                line-height: rem(21);
-                font-weight: $font-semibold;
-                margin: 0;
-            }
-            & .filter-icon {
-                @include flex(0 0 auto);
-                width: rem(16);
-                margin-right: rem(8);
-                & .usa-da-icon-filter {
-                    fill: lighten( $color-gray, 15%);
+            .sidebar-header_title{
+                display: flex;
+                flex-flow: inherit;
+                padding-top: rem(16);
+                padding-bottom: rem(16);
+                & .sidebar-title {
+                    @include h6;
+                    @include flex(1 1 auto);
+
+                    color: $color-gray;
+                    font-size: rem(19);
+                    line-height: rem(21);
+                    font-weight: $font-semibold;
+                    margin: 0;
+                }
+                & .filter-icon {
+                    @include flex(0 0 auto);
                     width: rem(16);
-                    margin-top: rem(5);
+                    margin-right: rem(8);
+                    & .usa-da-icon-filter {
+                        fill: lighten( $color-gray, 15%);
+                        width: rem(16);
+                        margin-top: rem(4);
+                    }
+                }
+
+            }
+            .sidebar-paragraph__container{
+                padding-bottom: rem(16);
+                display: flex;
+                flex-flow: row;
+                padding-right: rem(4);
+                .sidebar-paragraph {
+                    font-weight: $font-semibold;
+                    font-size: $font-size-14;
+                }
+                .tooltip-wrapper {
+                    display: inline-block;
+                    text-align: bottom;
```

Comment:
'bottom' is not an accepted value for text-align; this can be removed without affecting the appearance of the component

Reply:
done